app/src/main/java/chat/rocket/android/chatroom/ui/ChatRoomFragment.kt

```diff
@@ -23,6 +24,11 @@ import androidx.fragment.app.Fragment
 import androidx.recyclerview.widget.DefaultItemAnimator
 import androidx.recyclerview.widget.LinearLayoutManager
 import androidx.recyclerview.widget.RecyclerView
+import android.text.SpannableStringBuilder
+import android.view.*
```

Comment:
Don't use wildcard imports please.

Reply:
Sure! But can you tell me why it's the bad idea.

Reply:
There are many recommendations about why using wildcard is bad (read this [article](http://www.javadude.com/articles/importondemandisevil.html) for instance), in the other side some people advocates by using wildcard (there is even a recommendation for doing that in the Clean Code book).

In short, by not using the wildcard won't fail the build when we add the lint task here in the future (like we have for our Kotlin SDK).